.github/workflows/ci.yml

```diff
@@ -35,21 +35,12 @@ jobs:
                   tools: phpstan,flex
                   extensions: zip
 
-            - name: Get composer cache directory
-              id: composercache
-              run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
-
-            - name: Cache dependencies
-              uses: actions/cache@v4
+            - name: "Install dependencies with composer"
+              uses: "ramsey/composer-install@3.0.0"
               with:
-                  path: ${{ steps.composercache.outputs.dir }}
-                  key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.json') }}
-                  restore-keys: ${{ runner.os }}-composer-
-
-            - name: Install dependencies
-              run: composer install --prefer-dist
+                  composer-options: "--prefer-dist"
```

Comment:
This is now the default. This can be removed everywhere.

Reply:
Are you sure? I double checked the README yesterday, because I was unsure and it isn't mentioned explicitly

Reply:
Yes!

> Composer uses dist by default.

https://getcomposer.org/doc/03-cli.md#install-i

Reply:
Ah indeed, thanks for clarification